app/models/rules_factory.rb

```diff
@@ -4,10 +4,18 @@ def initialize(auction)
   end
 
   def create
+    eligibility = if auction.small_business?
```

Comment:
I am fine with sticking to eligiblity for now. I am guessing another term may emerge as we learn more about different types of auctions and how their rules are set